datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -66,6 +66,12 @@
     SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam("name") String name,
             @Valid SdxClusterRequest createSdxClusterRequest);
 
+    @GET
+    @Path("list")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "list SDX clusters", produces = MediaType.APPLICATION_JSON, nickname = "listSdx")
```

Comment:
Not an expert of Swagger def, but I would assume that `nickname = "listSdx"` should be something unique.